### PR TITLE
ticdc: add a new monitor metrics

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/monitor-ticdc.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:1
 - MySQL sink conflict detect duration percentile：每秒钟中 95%、99% 和 99.9% 的情况下，MySQL 写入冲突检测耗时。
 - MySQL sink worker load：TiCDC 节点中写 MySQL 线程的负载情况。
 
+**Changefeed** 面板新增重要指标：
```
Comment (ran-huang) [inline ticdc/monitor-ticdc.md:70]: 不需要提出是*新增*指标，请把 changefeed catch-up ETA 合并到上方的列表中，并把图片移到 L50 下方

Comment (asddongmen) [reply ticdc/monitor-ticdc.md:70]: 这个指标不存在上面的 Changefeed dashboard 中，如果要把上面那张图直接替换掉，工作量会比较大。
或许可以把新增去掉，只留下重要指标。

Comment (ran-huang) [reply ticdc/monitor-ticdc.md:70]: 能否调整下各个图片和列表的排列方式，在每个图片下方列出图上的几个 metrics 的文字描述。对用户来说，图片和文字更靠近会更易读一些。

Comment (asddongmen) [reply ticdc/monitor-ticdc.md:70]: @leoppro 应该咋写好点呢？

Comment (ran-huang) [reply ticdc/monitor-ticdc.md]: 看看这个可以不 f736ba0

**Review by leoppro — APPROVED** [review]

File: ticdc/monitor-ticdc.md [hunk 2]
```diff
@@ -59,14 +55,24 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:1
 - Exit error count：每分钟内导致同步中断的错误发生次数。
 - Changefeed checkpoint lag：同步任务上下游数据的进度差（以时间计算）。
 - Changefeed resolved ts lag：TiCDC 节点内部同步状态与上游的进度差（以时间计算）。
```
Comment (Oreoxmt) [inline ticdc/monitor-ticdc.md:57]:
```suggestion
- Exit error count/m：每分钟内导致同步中断的错误发生次数。
- Changefeed checkpoint lag：同步任务上下游数据的进度差（以时间计算）。
- Processor resolved ts lag：TiCDC 节点内部同步状态与上游的进度差（以时间计算）。
```
图中是 Processor reesolved ts lag

File: ticdc/monitor-ticdc.md [hunk 3]
```diff
@@ -59,14 +55,24 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:1
...
-- Flush sink duration：TiCDC 异步刷写数据入下游的耗时直方图。
-- Flush sink duration percentile：每秒钟中 95%、99% 和 99.9% 的情况下，TiCDC 异步刷写数据入下游所花费的时间。
+
+![TiCDC Dashboard - Changefeed metrics 2](/media/ticdc/ticdc-dashboard-changefeed-2.png)
+
 - Sink write duration：TiCDC 将一个事务的更改写到下游的耗时直方图。
 - Sink write duration percentile：每秒钟中 95%、99% 和 99.9% 的情况下，TiCDC 将一个事务的更改写到下游所花费的时间。
+- Flush sink duration：TiCDC 异步刷写数据入下游的耗时直方图。
```
Comment (Oreoxmt) [inline ticdc/monitor-ticdc.md:63]: 图中还有 sink write rows count/s 和 sink write batch size percentile 两个指标，文档中缺少这两个指标的说明

**Review by ran-huang — APPROVED** [review]

File: ticdc/monitor-ticdc.md (hunk 1 above)
Comment (asddongmen) [reply ticdc/monitor-ticdc.md]: 可以！

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**